bedrock/base/templates/includes/protocol/navigation/menu-firefox/browsers.html

```diff
@@ -0,0 +1,91 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set referral="?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=nav&amp;utm_content=firefox" %}
+
+<li class="mzp-c-menu-category mzp-has-drop-down mzp-js-expandable">
+  <a class="mzp-c-menu-title" href="{{ url('firefox.new') }}" aria-haspopup="true" aria-controls="mzp-c-menu-panel-browsers">{{ _('Browsers') }}</a>
+  <div class="mzp-c-menu-panel mzp-has-card" id="mzp-c-menu-panel-browsers">
+    <div class="mzp-c-menu-panel-container">
+      <button class="mzp-c-menu-button-close" type="button" aria-controls="mzp-c-menu-panel-browsers">{{ _('Close Browsers menu') }}</button>
+      <div class="mzp-c-menu-panel-content">
+        <ul>
+          <li>
+            <section class="mzp-c-menu-item mzp-has-icon">
+              <a class="mzp-c-menu-item-link" href="{{ url('firefox.new') }}" data-link-name="Firefox Desktop" data-link-type="nav" data-link-position="topnav" data-link-group="browsers">
+                <svg class="mzp-c-menu-item-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="20" viewBox="0 0 24 20"><g fill="none" fill-rule="evenodd" stroke="#000" stroke-linecap="round" transform="translate(1.77 1)"><rect width="18" height="12" x="1.225" stroke-linejoin="round" stroke-width="2" rx="2"/><path stroke-linejoin="round" stroke-width="2" d="M2.66666667,12 L17.7836298,12 C18.6444906,12 19.4087683,12.5508602 19.6809964,13.3675445 L20.347663,15.3675445 C20.6969586,16.4154311 20.1306386,17.5480711 19.082752,17.8973666 C18.878819,17.9653442 18.6652606,18 18.4502965,18 L2,18 C0.8954305,18 -3.80449182e-13,17.1045695 -3.80584453e-13,16 C-3.80610778e-13,15.7850358 0.034655758,15.5714774 0.102633404,15.3675445 L0.769300071,13.3675445 C1.04152816,12.5508602 1.80580585,12 2.66666667,12 Z"/><path stroke-width="2" d="M6.22514823,15 L14.2251482,15"/></g></svg>
+                <h4 class="mzp-c-menu-item-title">Firefox Desktop Browser</h4>
```

Comment:
Unwrapped string. Or are we treating this whole thing as an untranslated brand name?